packages/design-system/src/components/primitives/dnd/placement-indicator.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from "../../box";
+import { type Placement } from "./geometry-utils";
+
+const placementStyle = {
+  boxSizing: "content-box",
+  position: "absolute",
+  background: "#f531b3",
+  pointerEvents: "none",
+};
+
+export const PlacementIndicator = ({ placement }: { placement: Placement }) => {
+  const style =
```

Comment:
would probably put this into a function and use early return to avoid multiline ternaries and also to avoid cluttering the component